src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
             --enable-compression
 """
 
+helps['cdn endpoint rule'] = """
+type: group
+short-summary: Manage delivery rules for an endpoint.
+"""
+
+helps['cdn endpoint rule add'] = """
+type: command
+short-summary: Add a delivery rule to a CDN endpoint.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+  - name: --order
+    type: string
+    short-summary: Order of the rule.
+  - name: --match-variable
+    type: string
+    short-summary: Name of the match condition.
+  - name: --operator
+    type: string
+    short-summary: Operator of the match condition.
+  - name: --selector
+    type: string
+    short-summary: Selector of the match condition.
+  - name: --match-values
+    type: string
+    short-summary: Match values of the match condition (comma separated).
+  - name: --negate-condition
+    type: string
+    short-summary: Describes if this is negate condition or not.
+  - name: --transform
+    type: string
+    short-summary: Describes what transforms are applied before matching.
+  - name: --action-name
+    type: string
+    short-summary: Name of the action.
+  - name: --cache-behavior
+    type: string
+    short-summary: Caching behavior for the requests. Possible values are BypassCache, Override, SetIfMissing.
```

Comment:
You need to set `arg_type=get_enum_type(['BypassCache', 'Override', 'SetIfMissing'])